src/main/app/forms/models/claimantDetails.ts

```diff
@@ -1,6 +1,6 @@
 import { Serializable } from 'app/models/serializable'
 import { IsDefined, IsIn, IsOptional, MaxLength, ValidateIf } from 'class-validator'
-import { PartyTypes, PartyTypes as ClaimantTypes } from 'app/forms/models/partyTypes'
+import { PartyType, PartyType as ClaimantTypes } from 'app/common/partyType'
```

Comment:
Please use either the  `PartyType` or the alias `ClaimantTypes`. Prefer to name alias as singular `ClaimantType`.